index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="description" content="Quote-Generator">
+        <meta name="author" content="Mitiny-Mitiny">
+        <meta name="keywords" content="HTML, CSS, Javascript, Quotes">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Quote-Generator</title>
+        <link rel="stylesheet" href="mystyle.css"> 
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+    </head>
+
+    <body>
+        <div class="container">
+            <h1>
+                <i class="fa fa-quote-left"></i>
+                <span class="quote" id="quote">Genius is one percent inspiration and ninety-nine percent perspiration</span>
```

Comment:
At your own time , you can try to call the getNewQuote function when the page is loaded for the first time.
this will avoid kueka quote ya kwanza and that why all quotes will be coming from the API.